### DISCUSSION TITLE
godaddy and digital ocean

### DISCUSSION THREAD

**Discussion by hendrikjap** [Budibase/budibase#3853] [Help] [answered]
Hi guys,

I need some help. I have a custom domain with godaddy. I have set up subdomain and created A record to be directed to my digital ocean budibase droplet.

My installation is using docker droplet and then npm budibase cli (not 1 click deployment of budibase to DO). 
My godaddy domain already has SSL cert but not my DO droplet.
I have created budibase to default listen to port 80.
I have allowed sudo ufw allow to 80 and 443.

It's already more than 24 hours but I still can't connect to budibase using my custom domain. 

What should I check or do?
Do I need still need to reverse proxy?

Thanks!😊

**Comment by mjashanks** [comment, marked as answer]
Yes, you will still need to setup a reverse proxy for this! https://docs.budibase.com/self-hosting/hosting-settings/advanced-options/reverse-proxy

**Reply by hendrikjap** [reply]
I am amazed on how fast you guys reply. 

Btw, I need to add some tips for custom domain digital ocean deployment. Due to digital ocean firewall droplet, we need to sudo ufw allow some of the ports so that caddy can work properly.
